.gitignore

```diff
@@ -1,3 +1,4 @@
 public
 .DS_Store
-.hugo_build.lock
\ No newline at end of file
+.hugo_build.lock
+.idea
```

Comment:
why?

Reply:
Because, it works even if we wanted to work in Intellij.